src/ResultStore.jl

```diff
@@ -58,8 +58,8 @@ function create_dataset(start_year::Int, end_year::Int, n_reefs::Int, reps::Int)
 
     arr_size = (year_range, n_reefs, 2 * reps)
     
-    # Coral cover [% of total reef area]
-    cover = DataCube(
+    # Relative Coral cover [% of total reef area]
+    relative_cover = DataCube(
```

Comment:
Hey Dan, are we sure this is % relative to total reef area?

Reply:
In the RME data codes it appears that it is, though perhaps the comment should specify `[% of reef area]` as total might imply the entire study area. The variable name used to retrieve the data from the API is 'coral_pct'
![image](https://github.com/open-AIMS/ReefModEngine.jl/assets/156630392/01d7388a-f123-48bb-b6a1-bda33206a6ef)

Reply:
Hmmm...

There are conflicting definitions of what "total" refers to. Let me ask.

Reply:
I've updated the names to be `total_cover` and `total_taxa_cover`.

Reply:
This is fine with me, but noting it requires changes in ADRIA for the naming convention to be fully supported.